src/main/java/io/github/genomicdatainfrastructure/discovery/services/PackageShowMapper.java

```diff
@@ -51,13 +48,13 @@ public RetrievedDataset from(CkanPackage ckanPackage) {
                 .accessRights(value(ckanPackage.getAccessRights()))
                 .conformsTo(values(ckanPackage.getConformsTo()))
                 .provenance(ckanPackage.getProvenance())
-                .spatial(value(ckanPackage.getSpatialUri()))
+                .spatial(value(ckanPackage.getSpatial()))
```

Comment:
important: this is a bug on CKAN side, it should still be `spatial_uri`, because we did not rename the field in SOLR schema and other places. I will fix it right away there.

Reply:
For spatial_uri we get empty string from ckan, so it fails. You will fix that? Or should we go back for spatial_uri to be string?